torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -270,6 +272,12 @@ static inline bool treatSequenceAsTuple(PyObject* index) {
   if (THPVariable_Check(index)) {
     return false;
   }
+#ifdef USE_NUMPY
+  TORCH_CHECK(::torch::utils::is_numpy_available(), "Numpy is not available");
+  if (PyArray_CheckExact(index)) {
```

Comment:
Sorry, I edited the message. I think this should be
```suggestion
  if (::torch::utils::is_numpy_available() && PyArray_CheckExact(index)) {
```